nx/include/net/dlt.h

```diff
@@ -0,0 +1,1338 @@
+/*-
```

Comment:
Is there any reason to pull in this thing when literally nobody is going to use it?

Reply:
Dependency of bpf.h